model/src/lib.rs

```diff
@@ -113,7 +113,14 @@ impl OrderCreation {
 // See https://github.com/gnosis/gp-v2-contracts/blob/main/src/contracts/libraries/GPv2Encoding.sol
 impl OrderCreation {
     const ORDER_TYPE_HASH: [u8; 32] =
-        hex!("b71968fcf5e55b9c3370f2809d4078a4695be79dfa43e5aa1f2baa0a9b84f186");
+        hex!("b2b38b9dcbdeb41f7ad71dea9aed79fb47f7bbc3436576fe994b43d5b16ecdec");
+
+    // keccak256("sell")
+    const ORDER_KIND_SELL: [u8; 32] =
+        hex!("f3b277728b3fee749481eb3e0b3b48980dbbab78658fc419025cb16eee346775");
```

Comment:
As for the order type hash, I am open to computing this in a more high level way by actually doing the keccak but for now I  am keeping the exact hex string as it is in the contract to make it obviously correct.